manifests/p/Parsec/Parsec/150-84c/Parsec.Parsec.yaml

```diff
@@ -2,7 +2,7 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
 
 PackageIdentifier: Parsec.Parsec
-PackageVersion: "150-82a"
+PackageVersion: "150-84c"
```

Comment:
```suggestion
PackageVersion: "150-84b"
```
This must match Control Panel.

Reply:
![image](https://user-images.githubusercontent.com/11600822/182004092-e1575e6e-b5a0-4305-a94a-c53333e63f57.png)